portfolio/src/main/webapp/script.js

```diff
@@ -35,6 +35,34 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//*********************** PAGE HANDLING ***********************
+/**
+ * Displays the right sidebar
+ */
+let isSidebarOpen = false;
+function toggleSidebar(){
+    let button = document.getElementById("commentbutton");
+    let el = document.getElementById("sidebar");
+    if (isSidebarOpen == true){
```

Comment:
use triple equals (`===`) when comparing values. `==` tends to be ambiguous and has unintuitive results:
```
[] == '' // true
[] == false // true
{} == true // true
```
https://www.destroyallsoftware.com/talks/wat (start at 1:20).
If you use triple equals, all of these are false instead of true, which is what most developers expect.